pkg/app/server.go

```diff
@@ -195,21 +195,30 @@ func RunKubeStateMetrics(ctx context.Context, opts *options.Options, factories .
 	telemetryListenAddress := net.JoinHostPort(opts.TelemetryHost, strconv.Itoa(opts.TelemetryPort))
 	telemetryServer := http.Server{
 		Handler:           telemetryMux,
-		Addr:              telemetryListenAddress,
 		ReadHeaderTimeout: 5 * time.Second}
+	telemetryFlags := web.FlagConfig{
+		WebListenAddresses: &[]string{telemetryListenAddress},
+		WebSystemdSocket:   new(bool),
```

Comment:
Is this the socket-activated boot?

Reply:
This is https://github.com/prometheus/exporter-toolkit/pull/95